### DISCUSSION TITLE
More payment options

### DISCUSSION THREAD

**Discussion by Panda369420** [gameserverapp/Platform#727] [Suggestions]
If its possible to add a payment option for the payment to go to Nitrado, or to use paypal without needing the business side (for the API), im not familiar with Paypal like that and i have no clue what stripe is, all the money i get from any of my server's always goes back to the server in one way or another and the thought of needing to file taxes bothers me, i already pay taxes for my hard earned money from my job, but a way to not need the API would be nice and a way for players to choose stripe, paypal or Nitrado , or just not needing the API, i dont know if any of that is possible. I seen this was a suggestion for stripe and millie or something but said it was locked

**Comment by mvaessen** [comment]
Hi,

We're not able to mitigate PayPal's requirement about needing to have a business account. That's their policy.

Stripe is an alternative, and you can find more info about stripe on Google.

There are no plans to integrate anything nitrado related.

Closing this suggestions as it is mostly questions.
